.gitignore

```diff
@@ -58,3 +58,6 @@ typings/
 
 # UI5 tooling dist folder
 dist
+
+# WebStorm/IntelliJ IDE
+.idea
```

Comment:
näääh :-)

Reply:
But well, ok, let's keep it to avoid polluting the repo by webstorm users... *lol*